yapf/__init__.py

```diff
@@ -133,8 +133,12 @@ def main(argv):
     print('yapf {}'.format(__version__))
     return 0
 
+  style_config = args.style
+
   if args.style_help:
-    style.SetGlobalStyle(style.CreateStyleFromConfig(args.style))
+    if style_config is None and not args.no_local_style:
+        style_config = file_resources.GetDefaultStyleForDir(os.getcwd())
```

Comment:
This line needs to shift to the left two spaces.

Reply:
Sorry about that, fixed